### PR TITLE
[common] 1.6.0

### PR DESCRIPTION
**Pull request by onedr0p**
#### Special notes for your reviewer:

Closes https://github.com/k8s-at-home/charts/issues/173
Closes https://github.com/k8s-at-home/charts/issues/183

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

**Comment by onedr0p** [comment]
@bjw-s Update `template` references to `include`

**Comment by onedr0p** [comment]
@bjw-s `service.port.targetPort` and `service.port.name` are removed from values.yaml and defaulted in the service tpl 

https://github.com/k8s-at-home/charts/pull/178/commits/bb98d0cda5b97effa47a0d696832398339e1a40d

File: charts/common/values.yaml [hunk 1]
```diff
@@ -82,9 +84,11 @@ service:
   #   # Specify the default port information
   #   port:
   #     port: ""
```
Comment (bjw-s) [inline charts/common/values.yaml:86]:
```suggestion
  #     port: 
```
Small leftover from when we had the fluxv2 issues

**Comment by bjw-s** [comment]
I think this one is good to go into final review

**Pull request merged**